pyomo/contrib/mindtpy/global_outer_approximation.py

```diff
@@ -61,11 +61,6 @@ class MindtPy_OA_Solver(_MindtPyAlgorithm):
     - Global LP/NLP based branch-and-bound (GLP/NLP)
     - Regularized LP/NLP based branch-and-bound (RLP/NLP)
     - Feasibility pump (FP)
-
```

Comment:
Modify these comments to reflect what this class actually has (GOA). Also, is it the same class name as normal OA?